<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>声纹识别</string>
  </property>
  <widget class="QPlainTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>191</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="enrollButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>注册录音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="testButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>测试录音</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
